development/architecture/domain/domain-services.md

```diff
@@ -43,9 +43,9 @@ Here are some principles for implementing a Domain Service:
 ## Product domain services
 
 1. [Product update services](https://github.com/PrestaShop/PrestaShop/tree/1.7.8.x/src/Adapter/Product/Update)
-2. [Virtual product update services](https://github.com/PrestaShop/PrestaShop/blob/1.7.8.x/src/Adapter/Product/VirtualProduct/Update)
-2. [Combination update services](https://github.com/PrestaShop/PrestaShop/blob/1.7.8.x/src/Adapter/Product/Combination/Update)
-2. [Combination create services](https://github.com/PrestaShop/PrestaShop/blob/1.7.8.x/src/Adapter/Product/Combination/Create)
+2. [Virtual product update services](https://github.com/PrestaShop/PrestaShop/tree/1.7.8.x/src/Adapter/Product/VirtualProduct/Update)
```

Comment:
```suggestion
2. [Virtual product update services](https://github.com/PrestaShop/PrestaShop/tree/1.7.8.0/src/Adapter/Product/VirtualProduct/Update)
```

We agreed on linking to the tag version, right? It could be 1.7.8.0 (if the file exists in that version)

Reply:
yes, let me check on the whole repo